_reports/RJ722.md

```diff
@@ -0,0 +1,390 @@
+---
+categories:
+  - GSoC
+  - 2018
+  - StatusReport
+date: 2018-08-12
+report_year: 2018
+title: Rahul Jha
+student: Rahul Jha
+organisation: coala
+organisation_link : https://coala.io
+project: Improve Whitelisting Infrastructure for Vulture
+project_link: https://summerofcode.withgoogle.com/projects/#5620743946108928
+tarball: https://drive.google.com/file/d/12HgBb2Wdjl6Zzm7mDqCt13ke8VHj7_Bb/view
+mentors: >
+ [Jendrik Seipp](https://github.com/jendrikseipp) & [Abdeali Kothari](https://github.com/AbdealiJK) & [Florian Bruhin](https://github.com/The-Compiler)
+phase:
+ - Bonding : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/28
+ - Phase 1 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/29
+ - Phase 2 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/30
+ - Phase 3 : https://gitlab.com/coala/GSoC/GSoC-2018/milestones/31
+bio: >
+ Rahul is currently a junior, studying Electronics Engineering at Zakir Hussain
+ College of Engineering and Technology, Aligarh Muslim University. He loves the
+ ideology behind Free and Open Source Software and reads about the history of
+ this revolutionary movement in his leisure time.
+social:
+ - GitHub:
+   - username: RJ722
+   - link: https://github.com/RJ722
+ - GitLab:
+   - username: RJ722
+   - link: https://gitlab.com/RJ722
+ - Gitter:
+   - username: RJ722
+   - link: https://gitter.im/RJ722
+ - Openhub:
+   - username: RJ722
+   - link: https://openhub.net/accounts/RJ722
+email: rj722@protonmail.com
+blog: https://rj722.github.io/blog
+activity:
+ - 1:
```

Comment:
cEP & project definition commits at the top of the list

https://github.com/coala/projects/pull/571/commits/f551a2efef8fe7139c18c7bc92496ebd20ed7870